objects/src/constants.rs

```diff
@@ -35,10 +35,11 @@ pub const BATCH_OUTPUT_NOTES_TREE_DEPTH: u8 = 13;
 ///
 /// Because the tree used in a batch has fixed depth, and each note takes two leaves, the maximum
 /// number of notes is the number of leaves in the tree.
-pub const MAX_NOTES_PER_BATCH: usize = 2_usize.pow((BATCH_OUTPUT_NOTES_TREE_DEPTH - 1) as u32);
+pub const MAX_OUTPUT_NOTES_PER_BATCH: usize =
+    2_usize.pow((BATCH_OUTPUT_NOTES_TREE_DEPTH - 1) as u32);
 
 /// The maximum number of transaction in a single batch.
-pub const MAX_TRANSACTIONS_PER_BATCH: usize = MAX_NOTES_PER_BATCH / MAX_OUTPUT_NOTES_PER_TX;
+pub const MAX_TRANSACTIONS_PER_BATCH: usize = MAX_OUTPUT_NOTES_PER_BATCH / MAX_OUTPUT_NOTES_PER_TX;
```

Comment:
@bobbinth, please take a look into this constant. The result of the calculation is `1` (both `MAX_OUTPUT_NOTES_PER_BATCH` and `MAX_OUTPUT_NOTES_PER_TX` are `4096`).

Reply:
I'm not sure this constant is used anywhere. We should probably just remove it.